dart-runtime/lib/http_client.dart

```diff
@@ -67,23 +67,27 @@ class SdkgenHttpClient {
         packageInfo = await PackageInfo.fromPlatform();
       } catch (e) {}
 
-      var platform = {
+      final platform = {
         "os": Platform.operatingSystem,
-        "osVersion": Platform.operatingSystemVersion,
         "dartVersion": Platform.version,
         "appId": packageInfo?.packageName,
         "screenWidth": context == null ? 0 : MediaQuery.of(context).size.width,
         "screenHeight": context == null ? 0 : MediaQuery.of(context).size.height
       };
 
-      var deviceInfo = DeviceInfoPlugin();
+      final deviceInfo = DeviceInfoPlugin();
+
       if (Platform.isAndroid) {
-        var androidInfo = await deviceInfo.androidInfo;
-        platform["model"] = androidInfo.model;
-        platform["brand"] = androidInfo.brand;
+        final androidInfo = await deviceInfo.androidInfo;
+        platform["model"] = androidInfo.model; //Ex: SM-1234
+        platform["brand"] = androidInfo.brand; //Ex: Samsung
+        platform["osVersion"] = androidInfo.version.release; //10
```

Comment:
On Android-runtime this field is called just "version" (https://github.com/sdkgen/sdkgen/blob/master/android-runtime/runtime/src/main/java/io/sdkgen/runtime/SdkgenHttpClient.kt#L217-L220)

```suggestion
        platform["version"] = androidInfo.version.release; //10
```